builds/misc/packages.yaml

```diff
@@ -0,0 +1,269 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+pr: none
+variables:
+  REVISION: '1'
+  VSTS_WORK: '/home/vsts/work'
```

Comment:
Why not use one of the pre-defined variables like `BUILD_ARTIFACTSTAGINGDIRECTORY` instead of hard-coding this path?

Reply:
Using $HOME and the defaults instead